client/packages/requisitions/src/ResponseRequisition/DetailView/ReponseStats/ResponseStoreStats.tsx

```diff
@@ -44,33 +44,31 @@ export const ResponseStoreStats: React.FC<ResponseStoreStatsProps> = ({
           paddingBottom: 2,
         }}
       >
-        <>
```

Comment:
Not directly related to issue, but there was some unnecessary fragment blocks in here.